internal/cmd/auth_createapitokens.go

```diff
@@ -27,7 +29,14 @@ var createApiTokensCmd = &cobra.Command{
 			return err
 		}
 
-		tokenName := args[0]
+		tokenName := strings.TrimSpace(args[0])
+
+		validNameRegex := regexp.MustCompile("^[a-zA-Z0-9_-]+$")
```

Comment:
I think we should enforce the same rules for all entities' names.
The ones that we use for databases (on the server) seem good enough:

```go
func IsValidName(name string) bool {
	for _, r := range name {
		if !(unicode.IsDigit(r) || (unicode.IsLetter(r) && unicode.IsLower(r)) || r == '-') {
			return false
		}
	}
	return true
}
```

Let's use the same function here instead of regexes.
And since you're at it, please also validate database names before we send the request to the API.

Reply:
Thanks, I added the function and used it instead of regex.